GamaGameHub/GamaGameHub.Core/Models/Pagination/Pager.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GamaGameHub.Core.Models.Pagination
+{
+    public class Pager
+    {
+        public Pager() { }
+
+        public Pager(int totalItems, int page, int pageSize = 10) 
+        {
+            int totalPages = (int)Math.Ceiling((decimal)totalItems / (decimal)pageSize);
+            int currentPage = page;
+
+            int startPage = currentPage - 2;
```

Comment:
I think this can be rewritten as:
```
int startPage = currentPage - 2 <= 0 ? 1 : currentPage - 2;
int endPage = currentPage + 2 > totalPages ? totalPages : currentPage + 2;
if (endPage > 5)
{
    startPage = endPage - 4;
}